src/D7/FileTrait.php

```diff
@@ -11,10 +11,18 @@
  */
 trait FileTrait {
 
+  /**
+   * Files ids.
+   *
+   * @var array
+   */
+  static protected $fids = [];
```

Comment:
please remove `static`

variable and method names in all traits in this project must start with the relevant namespace
`$fileFids`